CIFAR10/train/fastai_pytorch.json

```diff
@@ -0,0 +1,11 @@
+{
+    "version": "v1.0",
+    "author": "fast.ai + students",
+    "authorEmail": "j@fast.ai",
+    "framework": "fastai / pytorch",
+    "codeURL": "https://github.com/fastai/imagenet-fast/",
```

Comment:
Please add the commit hash for reproducibility purposes.

Reply:
added the current master SHA1